resources/sql_scripts/update_scripts/snmp_msgfwd.sql

```diff
@@ -0,0 +1,45 @@
+-- Run this SQL update script if you already have a deployed CV Manager PostgreSQL database prior to the SNMP snmp_msgfwd_config table
+-- This file will create the 'snmp_msgfwd_type' and 'snmp_msgfwd_config' tables and add NTCIP 1218 and RSU 4.1 table names to the types table

```

Comment:
We create the two tables but I'm not seeing adding 1218 and 4.1 in here?

Reply:
I just added the comment lol to an old script. I can add this if you would like.

Reply:
Wait this isn't what I thought it was. We aren't adding those by name. We are adding ('rsuDsrcFwd'), ('rsuReceivedMsg'), ('rsuXmitMsgFwding'). Which are all of the relevant tables in those specs

Reply:
Let's make that a little clearer in the comment here so folks aren't confused then

Reply:
I added a comment to make it clearer